docs/listfilepages/globalfilestagpages/globalfilesprexxx.md

```diff
@@ -0,0 +1,2236 @@
+
+------------------------------------------------------------------------
+
+**<span id="predeitydomain"></span> Tag Name:** PREDEITYDOMAIN:x,y,y
+
+**Variables Used (x):** Number (The number of the deity's domains that must match).
+
+**Variables Used (y):** Text (Domain names).
+
+**What it does:**
+
+-   Sets requirements for the character's deity's domains.
+-   This tag applies to the domains of the PC's deity, and not the domains that the PC has selected for itself.
+
+**Example:**
+
+`     PREDEITYDOMAIN:1,Good,Law    `
+
+Character must have chosen of deity with either the "good" domain or the "law" domain.
+
+------------------------------------------------------------------------
+
+<span id="predomain"></span> \*\*\* Updated 5.11.7
+
+**Tag Name:** PREDOMAIN:x,y,y
+
+**Variables Used (x):** Number (The number of the deity's domains that must match).
+
+**Variables Used (y):** Text (Domain names or ANY).
+
+**What it does:**
+
+-   Set's requirements for a character's domains.
+-   This tag applies to the domains that the PC has selected for itself.
+
+**Example:**
+
+`     PREDOMAIN:1,Good,Law    `
+
+Character must have 1 of the two listed domains.
+
+`     PREDOMAIN:2,ANY    `
+
+Character must have 2 domains of any type.
+
+------------------------------------------------------------------------
+
+**<span id="predr"></span> Tag Name:** PREDR:x,y=z,y=z
+
+**Variables Used (x):** Number (The number of the DR conditions that must be met).
+
+**Variables Used (y):** Text (The type of DR).
+
+**Variables Used (z):** Number (Value the DR must be greater or equal to).
+
+**What it does:**
+
+Set's requirements for a character's Damage Resistance.
+
+**Examples:**
+
+`     PREDR:1,+1=10    `
+
+Must have DR of 10/+1 or greater (but DR Type must be +1).
+
+`     PREDR:1,-=10,+1=10,+2=10,+3=10,+4=10,+5=10,Silver=10    `
+
+Must have DR of 10 or greater of any type listed.
+
+------------------------------------------------------------------------
+
+**<span id="preequip"></span> Tag Name:** PREEQUIP:x,y,y
+
+**Variables Used (x):** Number (The number of items to be equipped)
+
+**Variables Used (y):** Text (Equipment name)
+
+**Variables Used (y):** TYPE=Text (Equipment type)
+
+**Variables Used (y):** WIELDCATEGORY=Text (Wield category of a weapon)
+
+**What it does:**
+
+-   This is used to determine if a character has a particular item(s) equipped.
+-   The percent symbol (%) may be used as a wild-card when including text in the list of parameters.
+-   Multiple TYPEs may be stacked by including them in a period delimited (.) list. (e.g. TYPE=Heavy.Armor for "Heavy Armor")
+
+**Examples:**
+
+`     PREEQUIP:1,Leather Armor    `
+
+Must have Leather Armor (only) equipped.
+
+`     PREEQUIP:1,Leather Armor%    `
+
+The "%" allows for items named Leather Armor (Masterwork), Leather Armor (+1) etc.
+
+`     PREEQUIP:1,TYPE=Armor    `
+
+Must have some type of armor equipped.
+
+`     PREEQUIP:2,TYPE=Armor,Sword (Long)%    `
```

Comment:
Maybe change this to "Sword (Short)%", I don't know of any set that had a 'Sword (Long)', best to stick with existing items whenever possible.